packages/ListBox/src/components/Filter/Filter.js

```diff
@@ -124,10 +87,10 @@ const Filter = React.forwardRef((props, ref) => {
   };
 
   React.useEffect(() => {
-    if (!state.isOpen) {
+    if (!isOpen) {
```

Comment:
I wonder if leaving state is better to understand the distinction if the value comes from the props or from the state? or doesn't really matter.

Reply:
Yeah, debatable.  In some cases, like this:

https://github.com/acl-services/paprika/pull/954/files#diff-b2903cb528031bc4a56d23248dfb6dc0093acec40ddf2141253c18151bb5335fR65

I think it's good to be explicit, but elsewhere I think it's a bit easier to read, and definitely easier to refactor, if it's destructured.